.shippable.yml

```diff
@@ -73,6 +73,7 @@ build:
     - _make PLATFORM=hikey
     - _make PLATFORM=hikey CFG_ARM64_core=y
     - _make PLATFORM=mediatek-mt8173 CFG_ARM64_core=y
+    - _make PLATFORM=mediatek-mt8516 CFG_ARM64_core=y
```

Comment:
Minor: for simplicity, and since `mediatek` is only ever built as 64-bit, how about first setting `CFG_ARM64_core ?= y` by default in `plat-mediatek/conf.mk`?

Reply:
Sure, it sounds good. I will change that.